cmd/up/activate.go

```diff
@@ -65,7 +65,7 @@ func (up *upContext) activate() error {
 	if err != nil {
 		return err
 	}
-	app, create, err := utils.GetApp(ctx, up.Dev, k8sClient, up.isRetry)
+	app, create, err := utils.GetApp(ctx, up.Dev, okteto.GetContext().Namespace, k8sClient, up.isRetry)
```

Comment:
Should we maintain the same way to get the namespace during the function? We already got it as `up.Namespace`. Is this one different?

Reply:
fixed in https://github.com/okteto/okteto/commit/b262d22a36a539a60335bee8182f6a2c348d7642